src/material/core/tokens/m2/mat/_divider.scss

```diff
@@ -0,0 +1,45 @@
+@use 'sass:map';
+@use '../../token-utils';
+@use '../../../theming/theming';
+@use '../../../style/sass-utils';
+
+// The prefix used to generate the fully qualified name for tokens in this file.
+$prefix: (mat, divider);
+
+// Tokens that can't be configured through Angular Material's current theming API,
+// but may be in a future version of the theming API.
+@function get-unthemable-tokens() {
+  @return (
+    thickness: 1px
```

Comment:
I think that we should call this `width`.

Reply:
I initially used `width`, but then I thought `width` isn't meaningful for vertical dividers...do you still think we should call it `width`?